package.json

```diff
@@ -8,14 +8,43 @@
     "name": "app",
     "main": "src/app.js",
     "dependencies": {
-      "bootstrap": "npm:bootstrap@4.0.0-alpha.2",
+      "bootstrap": "github:twbs/bootstrap@4.0.0-alpha.2",
```

Comment:
previously i was doing this `"files": [ "dist", "fonts", "js", "sass", "package.json" ],` [gist](https://gist.github.com/DavidStrada/bd33fdfdc876b4e48e2f3358fd6b66b8)